backend/app/services/pdf_analysis_service.py

```
@@ -0,0 +1,865 @@
+            for index, field in enumerate(sorted_fields):
+                field_data = self._process_field(field, text_elements, index)
+                result.append(field_data)
+            
+            self.logger.info(f"Successfully analyzed PDF: {len(result)} fields found")
+            return result
+            
+        except (PdfReadError, Exception) as e:
+            if isinstance(e, (NoFormFieldsError, FileNotFoundError)):
+                raise
+            
+            self.logger.error(f"PDF processing failed: {str(e)}")
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)